content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -59,6 +59,14 @@ To configure IPv4/IPv6 dual-stack, set dual-stack cluster network assignments:
       * `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` defaults to /24 for IPv4 and /64 for IPv6
    * kube-proxy:
       * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
+   * kubelet:
+      * when there is no `--cloud-provider` the administrator can pass a comma-separated pair
+        of IP addresses via `--node-ip` to manually configure dual-stack `.status.addresses`
+        for that Node.
+        If a Pod runs on that node in HostNetwork mode, the Pod reports these IP addresses in its
+        `.status.podIPs` field.
```

Comment:
The dual host-network podIPs applies to both cloud and non-cloud clusters. So, on non-cloud, you can specify dual `--node-ip`, and then you'll get dual host-network podIPs, and on cloud, the cloud provider can specify dual-stack `.status.addresses`, and then you'll get dual host-network podIPs.

Reply:
I thought that is assumed that all pods (hostNetwork too) will have dual-stack IPs  , this section is about configuring components, and cloud-provider users shouldn't have to touch anything ... do you feel this needs a better wording?

Reply:
If this is only about configuring components, and it's already assumed that nodes with dual-stack addresses will have dual-stack hostNetwork podIPs, then isn't the explanation about podIPs here irrelevant and unnecessary?

Reply:
it's based on the feedback from users, the workflow is typically an user reporting: "my pods are not dualstack".
Then you realize,  "ah, so only hostNetwork pods" are not dual stack and explain this kubelet thing, and why the order of the node-ip parameters matters.
I don't know what else we can do for making this clearer :shrug: